scala/scala_maven_import_external.bzl

```diff
@@ -208,7 +208,7 @@ jvm_import_external = repository_rule(
         "rule_name": attr.string(mandatory = True),
         "licenses": attr.string_list(mandatory = True, allow_empty = False),
         "jar_urls": attr.string_list(mandatory = True, allow_empty = False),
-        "jar_sha256": attr.string(),
+        "jar_sha256": attr.string(doc = "'jar_sha256' is deprecated. Please use 'artifact_sha256'"),
```

Comment:
@ittaiz is this what you meant by deprecation?

Reply:
I meant https://github.com/bazelbuild/bazel/issues/5654
TLDR:
```
set the default value of the attribute to None and then in the rule implementation check if the attribute is set to anything other than None and print a warning to the user
```

Reply:
done. string type can't get None